components/services/bat_ads/public/cpp/ads_client_mojo_bridge.h

```diff
@@ -80,6 +80,8 @@ class AdsClientMojoBridge
                       const ads::mojom::P2AEventType type,
                       const std::string& out_value) override;
 
+  void LogTrainingInstance(ads::mojom::TrainingInstancePtr instance) override;
```

Comment:
Is there a reason we append `Instance`?

Reply:
that was just to give some semantic meaning to the set of logs, but your suggestion is more consistent. Further I explained the semantics of this collection in a header comment, so will change